package.json

```diff
@@ -22,10 +22,10 @@
   "license": "MIT",
   "scripts": {
     "css": "npm-run-all --parallel css-lint* css-compile* --sequential css-prefix* css-minify*",
-    "css-compile": "node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 scss/karma.scss dist/karma.css",
+    "css-compile": "node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 scss/karma.scss dist/karma.css && node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 scss/karma-components.scss dist/karma-components.css",
```

Comment:
The default value for node-sass is --precision 5 is there a reason we are using --precision 6 here?

Reply:
i doubt it its been there since day 1, dont know what the default was then, will loose it

Reply:
Plus it will only affect the like of the grid col widths and thats if u r using the prebuilt css files which we dont